src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()"
+      method="post">
```

Comment:
Fix this on all code

```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      onsubmit="onSubmit()"
      method="post"
    >
```